themes/bootstrap3/templates/Auth/AbstractBase/login.phtml

```diff
@@ -1,21 +1,19 @@
 <? $account = $this->auth()->getManager(); ?>
 <? $sessionInitiator = $account->getSessionInitiator($this->serverUrl($this->url('myresearch-home'))); ?>
 <? if (!$sessionInitiator): // display default login form if no login URL provided ?>
-  <form method="post" class="form-horizontal" action="<?=$this->url('myresearch-home')?>" name="loginForm">
+  <form method="post" action="<?=$this->url('myresearch-home')?>" name="loginForm" class="form-login">
     <?=$this->auth()->getLoginFields()?>
     <input type="hidden" name="auth_method" value="<?=$account->getAuthMethod()?>">
     <input type="hidden" name="csrf" value="<?=$this->escapeHtmlAttr($account->getCsrfHash(true))?>" />
     <div class="form-group">
-      <div class="col-sm-offset-3 col-sm-9">
-        <? if ($account->supportsCreation()): ?>
-          <a class="btn btn-link createAccountLink" href="<?=$this->url('myresearch-account') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Create New Account')?></a>
-        <? endif; ?>
-        <input class="btn btn-primary" type="submit" name="processLogin" value="<?=$this->transEsc('Login')?>">
-        <? if ($account->supportsRecovery()): ?>
-          <br/>
-          <a class="btn btn-link" href="<?=$this->url('myresearch-recover') ?>?auth_method=<?=$account->getAuthMethod()?>"><?=$this->transEsc('Forgot Password')?></a>
-        <? endif; ?>
-      </div>
+      <? if ($account->supportsCreation()): ?>
```

Comment:
Do you think it looks strange having create / recover links to the left of the login button now that the whole block is left-aligned instead of centered?